src/numbers.js

```diff
@@ -1,45 +1,47 @@
-function add (a, b) {
-  // your code here
-}
+
+  const add = (a, b) => {
+    return a + b;
+  };
+
 
 function subtract (a, b) {
-  // your code here
+  return a-b;
 }
 
 function multiply (a, b) {
-  // your code here
+  return a*b;
 }
 
 function divide (a, b) {
-  // your code here
+  return a/b;
 }
 
 function power (a, b) {
-  // your code here
+  return a**b;
 }
 
 function round (a) {
-  // your code here
+  return Math.round(a);
 }
 
 function roundUp (a) {
-  // your code here
+  return Math.ceil(a);
 }
 
 function roundDown (a) {
-  // your code here
+  return Math.floor(a);
 }
 
 function absolute (a) {
-  // your code here
+  return Math.abs(a);
 }
 
 function quotient (a, b) {
-  // your code here
+  return ~~(a/b);
```

Comment:
```suggestion
  return Math.trunc(a / b);
```
Clever use of bitwise operators here, but it isnt very clear what this function is doing from the outset. If there is a simpler way of doing things that is more immediately obvious that is available, its generally better to go with that one